packages/frontend-2/components/viewer/saved-views/panel/views/Group.vue

```diff
@@ -169,6 +175,104 @@ const menuItems = computed((): LayoutMenuItem<MenuItems>[][] => [
   ]
 ])
 
+const onDragOver = (_event: DragEvent) => {
```

Comment:
the convention is that `_var` is only for unused variables, you're using `_event` everywhere